tests/testthat/helper.R

```diff
@@ -0,0 +1,85 @@
+if (!requireNamespace("RBackend", quietly = TRUE)) {
+  stop("RBackend is needed to run the tests. Please install it.", call. = FALSE)
```

Comment:
I think the greatest service we can do for the extension is removing this, and making the tests not rely on the backend :). testwhat depends on the backend for almost all its tests, but IMO that was a mistake. This is especially apparent for your SCTs here, which don't need any code executed.

commented on `test_it` to discuss how to remove the backend